src/containers/navigation/index.tsx

```diff
@@ -0,0 +1,37 @@
+import Helper from 'containers/guide/helper';
+import GuideSwitcher from 'containers/guide/switcher';
+import LanguageSelector from 'containers/navigation/language-selector';
+import Menu from 'containers/navigation/menu';
+
+import News from './news';
+const HELPER_ID = 'menu-categories';
+
+const AppTools = () => (
+  <div className="absolute top-2 left-0 z-50 flex w-[540px] items-center justify-between space-x-4 rounded-full bg-brand-800 py-2 px-8 shadow-md print:hidden">
+    <Menu />
+
+    {process.env.NEXT_PUBLIC_VERCEL_ENV === 'development' && <News />}
+    <Helper
+      className={{
+        button: HELPER_ID ? '-bottom-10 -right-1.5 z-[20]' : 'hidden',
+        tooltip: 'w-fit-content',
+      }}
+      tooltipPosition={{ top: -40, left: -50 }}
+      message="language selector"
+    >
+      <LanguageSelector />
+    </Helper>
+    <Helper
```

Comment:
Helpers on green color (brand) doesn't fit very well. Maybe we can ask design to help us on it:
![Screenshot 2023-12-28 at 11 08 33](https://github.com/Vizzuality/mangrove-atlas/assets/51995866/6c7f302d-067a-44ad-818b-deff1bb952ee)